IdentityCore/src/cache/accessor/MSIDDefaultTokenCacheAccessor.m

```diff
@@ -687,7 +691,7 @@ - (BOOL)saveAccessTokenWithConfiguration:(MSIDConfiguration *)configuration
     // Delete access tokens with intersecting scopes
     MSIDDefaultCredentialCacheQuery *query = [MSIDDefaultCredentialCacheQuery new];
     query.homeAccountId = accessToken.accountIdentifier.homeAccountId;
-    query.environment = [[accessToken.authority cacheUrlWithContext:context] msidHostWithPortIfNecessary];
+    query.environment = [[accessToken.authority cacheUrlWithContext:context] msidHostWithPortIfNecessary];;
```

Comment:
nit: extra semicolon

Reply:
Dean, it's very helpful to leave suggestions for these sorts of comments - then Rohit can resolve it in one click